docs/marketplace/dev/feature-integration-guides/202108.0/glue/marketplace-merchant-feature-integration.md

```diff
@@ -57,13 +57,19 @@ Make sure that the following changes have been applied in transfer objects:
 | RestOrdersAttributesTransfer.merchantReferences | property | Created | src/Generated/Shared/Transfer/RestOrdersAttributesTransfer |
 | RestOrderDetailsAttributesTransfer.merchantReferences | property | Created | src/Generated/Shared/Transfer/RestOrderDetailsAttributesTransfer |
 | RestOrderItemsAttributesTransfer.merchantReference | property | Created | src/Generated/Shared/Transfer/RestOrderItemsAttributesTransfer |
-| RestErrorMessageTransfer | object | Created | src/Generated/Shared/Transfer/RestErrorMessageTransfer |
```

Comment:
why was it removed? Features which this one depends on already contain it?

Reply:
is supposed, that `RestErrorMessageTransfer ` is same for any rest module. And this one comes from Rest API Core (glue) right?
@ilyakubanov